tests/compose/fixtures/extends/common-env-labels-ulimits.yml

```diff
@@ -0,0 +1,13 @@
+web:
+  extends:
+    file: common.yml
+    service: web
+  environment:
+    - FOO=2
+    - BAZ=3
+  labels: ['label=one']
+  ulimits:
```

Comment:
`ulimites` is not supported yet. See https://github.com/vmware/vic-product/blob/master/docs/user_doc/vic_app_dev/docker_compose_file_options.md for more info about the syntax that is not supported.